tests/integration-tests-isolated/pom.xml

```diff
@@ -302,93 +133,59 @@
          </exclusions>
       </dependency>
       <dependency>
-         <groupId>org.apache.directory.server</groupId>
-         <artifactId>apacheds-server-annotations</artifactId>
-         <version>${directory-version}</version>
+         <groupId>org.apache.directory.api</groupId>
+         <artifactId>api-ldap-model</artifactId>
+         <version>2.0.0.AM1</version>
```

Comment:
We didnt have a direct dep on this before, so presumably it was transitive? I'd leave it that way rather than hard code a version; add a version prop or remove?